source/patterns/@aws-solutions-constructs/aws-s3-stepfunctions/lib/index.ts

```diff
@@ -34,7 +34,7 @@ export interface S3ToStepfunctionsProps {
   /**
    * Optional user provided props to override the default props for the S3 Bucket.
```

Comment:
Here's where we need a comment that the events feature must already be turned on for an existing bucket as this construct can't do it from an IBucket (I assume...)

And in the README.md